exchange/bitswap/workers.go

```diff
@@ -89,7 +89,7 @@ func (bs *Bitswap) provideWorker(px process.Process) {
 		defer cancel()
 
 		if err := bs.network.Provide(ctx, k); err != nil {
-			log.Error(err)
+			//log.Error(err)
```

Comment:
not sure this shouldnt be logged at all. are you trying to silence a particular error?